geodiff/tests/test_changeset_utils.cpp

```diff
@@ -149,6 +153,277 @@ TEST( ChangesetUtils, test_hex_conversion )
   EXPECT_EQ( hex2bin( "41ff" ), "A\xff" );
 }
 
+
+bool concatChangesetsSqlite3session( const std::string &A, const std::string &B, const std::string &out )
```

Comment:
so for tests we need sqlite with session and for release we do not need it anymore?

Reply:
removed